### ISSUE TITLE
`skywire-cli` proxy does not fetch proxy

### ISSUE THREAD

**Issue by jdknives** [skycoin/skywire#1543]
**Describe the bug**

Running `skywire-cli proxy list` fails to fetch the list of available SOCKS proxy servers on `develop` branch. 

The visor logs show:

```
[2023-05-08T20:44:21.550496+02:00] ERROR [visor]: Error getting public vpn servers: no service of type proxy registered
```

Meanwhile there are multiple proxy servers registered in the service discovery. 

**Environment information:**

- OS: MacOS Ventura
- Platform: Darwin 22.3.0 arm64

**Steps to Reproduce**

1. Clone `skycoin/skywire`
2. Checkout `develop`
3. Run `skywire-cli conf gen`
4. Run `skywire-visor -c skywire-config.json
5. Run `skywire-cli proxy start -k $(skywire-cli proxy list -n1 | cut -d ':' -f1)` to fetch the first proxy

**Comment by jdknives** [comment]
This issue so far can only be reproduced on MacOS with above setup. Ubuntu Linux with Go `1.18.1` works smoothly.

**Comment by 0pcom** [comment]
fixed in #1546
